addons/account/tests/test_account_move_reconcile.py

```diff
@@ -1901,6 +1901,7 @@ def test_reconcile_special_mexican_workflow_1(self):
         # 3th reconciliation inv1 + pay1
         self.assert_invoice_outstanding_reconciled_widget(inv2, {
             refund1.id: 28.46,
+            res['partials'].exchange_move_id.id: 6.34,
```

Comment:
https://github.com/odoo/odoo/blob/b18912d691527469045bba46b764e3e4761b1aae/addons/account/tests/test_account_move_reconcile.py#L1883-L1889

Given this, I would have expected the exchange move to be displayed only on `refund1`, like asserted here
https://github.com/odoo/odoo/blob/b18912d691527469045bba46b764e3e4761b1aae/addons/account/tests/test_account_move_reconcile.py#L1894-L1898

Reply:
it's really dubious indeed. The partial line display is a debit, it should be used only against a credit on the receivable account, so only on the refund :man_shrugging:

Reply:
This missing flush is blocking one of my branches.  Please fix it.